gnomad/utils/filtering.py

```diff
@@ -443,3 +443,58 @@ def filter_y_nonpar(
         if isinstance(t, hl.Table)
         else t.filter_rows(non_par_expr)
     )
+
+
+def filter_by_numeric_expr_range(
+    t: Union[hl.MatrixTable, hl.Table],
+    filter_expr: hl.NumericExpression,
+    filter_range: tuple,
+    keep_between: bool = True,
+    inclusive: bool = True,
+) -> Union[hl.MatrixTable, hl.Table]:
+    """
+    Filter rows in the Table/MatrixTable based on the range of a numeric expression.
+
+    :param t: Input Table/MatrixTable.
+    :param filter_expr: Expression to which filter should be applied.
+    :param filter_range: Range of values to which filter should be applied
+    :param keep_between: Whether or not to keep the values between the range.
```

Comment:
```suggestion
    :param keep_between: Whether to keep the values between `filter_range` instead of keeping values outside `filter_range`. Default is True.
```